src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let stringOfArray = `${this[0]}`;
```

Comment:
Краще не називати змінні структурами данних
```suggestion
    let joinedСhars = `${this[0]}`;
```